crypto/rand/sysrand.c

```diff
@@ -88,21 +92,16 @@ int GFp_sysrand_chunk(void *out, size_t requested) {
 #endif
 #endif
 
-
-/* http://man7.org/linux/man-pages/man2/getrandom.2.html: "Calling
- * getrandom() to read /dev/urandom for small values (<= 256) of buflen is
- * the preferred mode of usage." */
-const size_t GFp_sysrand_chunk_max_len = 256;
-
 int GFp_sysrand_chunk(void *out, size_t requested) {
-  assert(requested <= GFp_sysrand_chunk_max_len);
-  if (syscall(SYS_getrandom, out, requested, 0u) < 0) {
-    if (errno == ENOSYS) {
-      return -1;
+  int r = syscall(SYS_getrandom, out, requested, 0u);
```

Comment:
I forgot that C's `long` is different on Windows than on other platforms. I pushed a fix for it: dcba8a845f645a4676eeaa1d57de827fce8aa94e